### PR TITLE
fix: hard code eth-flow contract addresses

### PR DESCRIPTION
**Pull request by shoom3301**
# Summary

Since eth-flow contract address is static and unlikely will change in the future, to keep the code simple I hardcoded addresses.
There is also an issue with resolving version of `@cowprotocol/ethflowcontract`, sometimes it returns outdated files locally.

  # To Test

Eth-flow should work

### CODE REVIEW

File: libs/common-const/src/common.ts [hunk 1]
```diff
@@ -52,19 +48,12 @@ export const APP_TITLE = 'CoW Swap | The smartest way to trade cryptocurrencies'
...
-    [SupportedChainId.SEPOLIA]: EthFlowProd[SupportedChainId.SEPOLIA].address,
-  },
-  barn: {
-    [SupportedChainId.MAINNET]: EthFlowBarn[SupportedChainId.MAINNET].address,
-    [SupportedChainId.GNOSIS_CHAIN]: EthFlowBarn[SupportedChainId.GNOSIS_CHAIN].address,
-    [SupportedChainId.GOERLI]: EthFlowBarn[SupportedChainId.GOERLI].address,
-    [SupportedChainId.SEPOLIA]: EthFlowBarn[SupportedChainId.SEPOLIA].address,
-  },
+  prod: mapAddressToSupportedNetworks(COWSWAP_ETHFLOW_CONTRACT_PROD),
+  barn: mapAddressToSupportedNetworks(COWSWAP_ETHFLOW_CONTRACT_BARN),
```
Comment (alfetopito) [inline libs/common-const/src/common.ts:56]: Not 100% sure.
Right now the addresses are the same, but it's not guaranteed.
See [what Federico said](https://cowservices.slack.com/archives/C049GSMCTMW/p1671450797834959?thread_ts=1671449420.113069&cid=C049GSMCTMW)

Comment (shoom3301) [reply libs/common-const/src/common.ts:56]: Yeah, I understand.
For some reason, locally, yarn wrongly resolves `@cowprotocol/ethflowcontract` package:
https://cowservices.slack.com/archives/C0361CDG8GP/p1704727878010159

I have no idea why it's happening, but I also faced it before and can't reproduce now.
The only way I see to solve it - hardcoding

Comment (alfetopito) [reply libs/common-const/src/common.ts:56]: Maybe it has something to do with yarn.lock?

**Review by alfetopito — APPROVED** [review]
As I mentioned in the comment, I'm not very found of this solution, but it's blocking.
So, let's get it merged and revisit later when we add new networks.

File: libs/common-const/src/common.ts (hunk 1 above)
Comment (shoom3301) [reply libs/common-const/src/common.ts:56]: As far as I can see it contains proper data

**Comment by alfetopito** [comment]
Sadly, this broke Sepolia eth-flow https://cowservices.slack.com/archives/C0361CDD1FZ/p1704815798996489

I'll revert this change and try to fix the issue some other way.

**Pull request merged**